spec/operators/max-spec.ts

```diff
@@ -184,42 +184,42 @@ describe('Observable.prototype.max', () => {
     expectSubscriptions(e1.subscriptions).toBe(e1subs);
   });
 
-  it('should handle a constant predicate on observable with many values', () => {
-    const e1 = hot('-x-^-a-b-c-d-e-f-g-|');
-    const e1subs =    '^               !';
-    const expected =  '----------------(w|)';
+  it('should handle a predicate on observable with many values', () => {
```

Comment:
is this test case needed still? originally it was testing predicate returns constant value (which was incorrect assessment), testing execution of predicate is being covered by line 200 (https://github.com/ReactiveX/rxjs/pull/1923/files#diff-1e06648035d9126f1295caf9df494e1cR200) .